lib/LedgerSMB/Scripts/recon.pm

```diff
@@ -275,34 +276,23 @@ sub _display_report {
     $recon->{out_of_balance} = $recon->{their_total} - $recon->{our_total};
         $recon->{cleared_total} = $recon->{cleared_total}->to_output(money => 1);
         $recon->{outstanding_total} = $recon->{outstanding_total}->to_output(money => 1);
-        $recon->{mismatch_our_debits} =
-                $recon->{mismatch_our_debits}->to_output(money => 1);
-        $recon->{mismatch_our_credits} =
-        $recon->{mismatch_our_credits}->to_output(money => 1);
-        $recon->{mismatch_their_debits} =
-        $recon->{mismatch_their_debits}->to_output(money => 1);
-        $recon->{mismatch_their_credits} =
-        $recon->{mismatch_their_credits}->to_output(money => 1);
-        $recon->{statement_gl_calc} =
-        $recon->{statement_gl_calc}->to_output(money => 1);
-        $recon->{total_cleared_debits} =
-                $recon->{total_cleared_debits}->to_output(money => 1);
-        $recon->{total_cleared_credits} =
-                $recon->{total_cleared_credits}->to_output(money => 1);
-        $recon->{total_uncleared_debits} =
-                $recon->{total_uncleared_debits}->to_output(money => 1);
-        $recon->{total_uncleared_credits} =
-                $recon->{total_uncleared_credits}->to_output(money => 1);
+    $recon->{mismatch_our_debits} = $recon->{mismatch_our_debits}->to_output(money => 1);
+    $recon->{mismatch_our_credits} = $recon->{mismatch_our_credits}->to_output(money => 1);
+    $recon->{mismatch_their_debits} = $recon->{mismatch_their_debits}->to_output(money => 1);
+    $recon->{mismatch_their_credits} = $recon->{mismatch_their_credits}->to_output(money => 1);
+    $recon->{statement_gl_calc} = $recon->{statement_gl_calc}->to_output(money => 1);
+    $recon->{total_cleared_debits} = $recon->{total_cleared_debits}->to_output(money => 1);
+    $recon->{total_cleared_credits} = $recon->{total_cleared_credits}->to_output(money => 1);
+    $recon->{total_uncleared_debits} = $recon->{total_uncleared_debits}->to_output(money => 1);
+    $recon->{total_uncleared_credits} = $recon->{total_uncleared_credits}->to_output(money => 1);
```

Comment:
@ylavoie if you're refactoring this anyway, can we do something to make it more concise, like:
```perl
    for my $amt_name (qw/ mismatch_our_ mismatch_their total_cleared_ total_uncleared_ /) {
      for my $bal_type (qw/ credits debits/) {
         $recon->{"$amt_name$bal_type"} = $recon->{"$amt_name$bal_type"}->to_output(money=>1);
      }
   }
```